matomo/templates/cronjob-archive.yaml

```diff
@@ -46,7 +46,7 @@ spec:
               - secretRef:
                   name: {{ include "matomo.fullname" . }}-envs
             args: ['su', 'www-data', '-s', '/bin/sh', '-c', '/usr/local/bin/php /var/www/html/console core:archive --url={{ include "matomo.fullname" . }}']
-          {{- with .Values.resources }}
+          {{- with .Values.cronjobs.archive.resources }}
             resources:
               {{ toYaml . | indent 14 | trim}}
           {{- end }}
```

Comment:
Fehlt hier nicht noch der persistence Kram?

Reply:
Nein, der archive schreibt in die Datenbank und benötigt keine persistence, da die DB-Connection per environment variablen hergestellt wird